src/components/AddBook.js

```diff
@@ -1,16 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
 
-const AddBook = () => (
-  <div className="addbook-content">
-    <h4>ADD NEW BOOK</h4>
-    <div className="add-book-btns">
-      <form>
-        <input type="text" />
-        <input type="text" />
-        <input type="submit" value="ADD BOOK" />
+const AddBook = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
```

Comment:
- [ ] It will be great to add both the title and the author to your local state. I can see that you have only added the title. `author` is also a major requirement before your project can be approved. Remember your onChange function should also consider the author in your state.